src/components/Verse/VerseText.tsx

```diff
@@ -33,12 +34,13 @@ const VerseText = ({ words, isReadingMode = false }: VerseTextProps) => {
         className={classNames(
           styles.verseTextContainer,
           styles[`quran-font-size-${quranTextFontScale}`],
+          { [styles.specialLayoutContainer]: hasSpecialLayout },
         )}
       >
         <div
           className={classNames(styles.verseText, {
             [styles.verseTextWrap]: !isReadingMode,
-            [styles.bigText]: isReadingMode && quranTextFontScale > 3,
+            [styles.specialLayout]: hasSpecialLayout,
```

Comment:
ah, saw that you changed it here. Not a big fan of the word "Special" cause it's vague. maybe largeQuranText and largeQuranTextLayout?